### DISCUSSION TITLE
PyPI dify-client Code Outdated and Mismatched

### DISCUSSION THREAD

**Discussion by pingfury108** [langgenius/dify#9770] [General]
Could there be an issue with the updates for the Python SDK (dify-client) on PyPI.org? The last update was in 2023, and the code doesn't seem to match the version on GitHub.

![image](https://github.com/user-attachments/assets/989d08b6-e8c4-40d0-841c-affb300eacbf)

**Comment by odicis** [comment]
Same for the [nodejs client](https://www.npmjs.com/package/dify-client) on npm, which is 6 month old. There were some updates 3 months ago in the [source code](https://github.com/langgenius/dify/tree/main/sdks/nodejs-client). Some features are missing, in particular the WorkflowClient.
Please release the new versions of the clients.
